tests/lib/common.py

```diff
@@ -186,3 +189,30 @@ def handle_cleanup_input(msg):
         elif i == "d":
             pdb.set_trace()
             break
+
+
+def get_unpack(url, dst_file, unpack_folder):
+    """
+    Download a file and unpack it in given folder
```

Comment:
(nit) this will only unpack .tar.gz files.

Reply:
Yep lets's keep this for later.